Test/ServerTelemetryChannel.Test/Shared.Tests/Implementation/BackoffLogicManagerTest.cs

```diff
@@ -0,0 +1,248 @@
+﻿namespace Microsoft.ApplicationInsights.WindowsServer.Channel.Implementation
+{
+    using System;
+#if !NET40
+    using System.Diagnostics.Tracing;
+#endif
+
+    using System.Globalization;
+    using System.Net;
+    using System.Linq;
+
+    using Microsoft.ApplicationInsights.WindowsServer.Channel.Helpers;
+
+#if NET40
+    using Microsoft.Diagnostics.Tracing;
+#endif
+
+    using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+    using Microsoft.ApplicationInsights.Channel;
+    using Microsoft.ApplicationInsights.Channel.Implementation;
+    using Microsoft.ApplicationInsights.Extensibility.Implementation;
+    using Microsoft.ApplicationInsights.Web.TestFramework;
+    using Microsoft.ApplicationInsights.WindowsServer.TelemetryChannel.Helpers;
+    using Microsoft.ApplicationInsights.WindowsServer.TelemetryChannel.Implementation;
+    
+    using Assert = Xunit.Assert;
+
+    public class BackoffLogicManagerTest
+    {
+        [TestClass]
+        public class GetBackendResponse
+        {
+            public void ReturnNullIfArgumentIsNull()
+            {
+                var manager = new BackoffLogicManager(TimeSpan.Zero);
+                Assert.Null(manager.GetBackendResponse(null));
```

Comment:
These appear to be tests but they are not marked with the [TestMethod] attribute, why is that?